### ISSUE TITLE
can not use `status` cmd.

### ISSUE THREAD

**Issue by jinleileiking** [dbcli/mycli#223]
```
mysql uWFL9Wg6159@10.36.253.127:yingjian_duin> status
(1064, u"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'status' at line 1")
mysql uWFL9Wg6159@10.36.253.127:yingjian_duin> status;
(1064, u"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'status' at line 1")
```

**Comment by tsroten** [comment]
@jinleileiking Thanks for using mycli!

mycli doesn't currently support the `status` command. Let me take a look and see if it's something we could add.

**Comment by jinleileiking** [comment]
:+1:

**Comment by amjith** [comment]
Released as part of 1.6.0. Please upgrade: 

```
pip install -U mycli 
```
